src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -186,6 +206,19 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
  * Notifies delegate of event reports from the MTRDevice
  *
  * @param eventReport  An array of response-value objects as described in MTRDeviceResponseHandler
+ *
+ *                In addition to the MTREventPathKey and MTRDataKey containing the path and event values, eventReport also contains
+ * these keys:
+ *
+ *                MTREventNumberKey : NSNumber-wrapped uint64_t value. Monotonically increasing, but consecutive events may not have
+ * consecutive numbers.
```

Comment:
The indent here is odd.

But also, I think the assumption should be that if events are not consecutive then you might have missed some events, whether because you actually missed them or because there was a device reboot and you didn't re-subscribe soon enough...  Worth documenting that clearly.

Reply:
Changed this to say "Monotonically increasing, and consecutive event reports should have consecutive numbers unless device reboots, or if events are lost.